src/server/api/endpoints/users/recommendation.ts

```diff
@@ -15,44 +17,77 @@ export const meta = {
 };
 
 export default (params: any, me: ILocalUser) => new Promise(async (res, rej) => {
-	// Get 'limit' parameter
-	const [limit = 10, limitErr] = $.num.optional.range(1, 100).get(params.limit);
-	if (limitErr) return rej('invalid limit param');
-
-	// Get 'offset' parameter
-	const [offset = 0, offsetErr] = $.num.optional.min(0).get(params.offset);
-	if (offsetErr) return rej('invalid offset param');
-
-	// ID list of the user itself and other users who the user follows
-	const followingIds = await getFriendIds(me._id);
-
-	// ミュートしているユーザーを取得
-	const mutedUserIds = (await Mute.find({
-		muterId: me._id
-	})).map(m => m.muteeId);
-
-	const users = await User
-		.find({
-			_id: {
-				$nin: followingIds.concat(mutedUserIds)
+	var external = config.user_recommendation.external
+
+	if (external) {
```

Comment:
You should make this expression one line.

*Existing Codes Example:*
https://github.com/syuilo/misskey/blob/3220d69a6930151f33928b5d789150aacc4bc382/src/services/drive/add-file.ts#L36-L39

Reply:
Done. Thanks.